azure/v12.0.0/README.md

```diff
@@ -0,0 +1,80 @@
+## :zap:  Giant Swarm Release 12.0.0 for Azure :zap:
+
+This is the first release to support kubernetes 1.17 on Azure.
+
+As of this release NGINX Ingress Controller App is optional and not pre-installed component on Azure.
+This allows NGINX App installations to be managed independently from the base platform lifecycle. It is both benefit but also new responsibility to keep NGINX App installations up-to-date separately from rest of the cluster.
+Making NGINX optional enables use of other ingress controller alternatives without wasting resources where NGINX is not the preferred option.
+Upgrading existing tenant clusters with pre-installed NGINX will leave NGINX unchanged. Existing NGINX App custom resources will still have `giantswarm.io/managed-by: cluster-operator` label, but it will be ignored. The label will be cleaned up at a later point after all tenant clusters have been upgraded and Azure platform releases older than v12.0.0 archived.
+
+Apart from these important changes, there have been a high number of changes under the hood, including a few important
+security fixes coming from upstream projects we upgraded.
+
+This release also brings some technical changes that are needed to prepare the upcoming Cluster API release with
+Node Pools support.
+
+### Kubernetes v1.17.9
+
+- Updated from v1.16.12 -
+changelog since [v1.17.8](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1178),
+since [v1.17.7](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1177),
+since [v1.17.6](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1176),
+since [v1.17.5](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1175),
+since [v1.17.4](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1174),
+since [v1.17.3](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1173),
+since [v1.17.2](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1172),
+since [v1.17.1](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1171) and
+since [v1.17.0](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changes)
```

Comment:
I don't think we should include all changelogs here, I don't think anyone would read them. Also it's same file so maybe we can leave `[v1.17.0](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changes)` and that would be enough because you can scroll up to see all changelogs.

Does any of this apply here:
- Cloud Provider Labels reach General Availability: Added as a beta feature way back in v1.2, v1.17 sees the general availability of cloud provider labels.
- Volume Snapshot Moves to Beta: The Kubernetes Volume Snapshot feature is now beta in Kubernetes v1.17. It was introduced as alpha in Kubernetes v1.12, with a second alpha with breaking changes in Kubernetes v1.13.
- CSI Migration Beta: The Kubernetes in-tree storage plugin to Container Storage Interface (CSI) migration infrastructure is now beta in Kubernetes v1.17. CSI migration was introduced as alpha in Kubernetes v1.14.